src/main/scala/scorex/core/consensus/History.scala

```diff
@@ -104,6 +104,7 @@ object History {
     val Younger = Value(2)
     val Older = Value(3)
     val Nonsense = Value(4)
```

Comment:
Add comments, what is the correct response for what king of situation. Where should we return Unknown, where should we return Nonsense?

Reply:
Unknown is initial state, Nonsense is when we think other peer is going nuts. Not sure, maybe some additional types are needed.